frontend/src/tabs/nfts/NFTs.js

```diff
@@ -70,10 +102,18 @@ const NFTs = (props) => {
   // TODO: Minted nfts view w/ non owned nfts
   const [myNFTs, setMyNFTs] = React.useState([]);
   const [allNFTs, setAllNFTs] = React.useState([]);
+  const [myNftPagination, setMyNftPagination] = useState({
+    pageLength: 10,
+    page: 1
+  });
+  const [allNftPagination, setAllNftPagination] = useState({
+    pageLength: 10,
+    page: 1
+  });
 
   const retrieveMyNFTById = async (tokenId) => {
     try {
-      let getNFTBtTokenId = `get-nft?tokenId=${tokenId}`;
+      let getNFTBtTokenId = `get-nfts?tokenId=${tokenId}`;
```

Comment:
this should still be `get-nft` because this only retrieves the one nft

Reply:
Alright that works.
Thanks for the feebback